snuba/query/allocation_policies/concurrent_rate_limit.py

```diff
@@ -31,7 +31,7 @@
 )
 
 
-class BaseConcurrentRateLimitAllocationPolicy(AllocationPolicy):
```

Comment:
This does not need to be changed. The base is a concurrent limit policy, has nothing to do with customers

Reply:
oops! the find-replace changed that